composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^7.0",
-        "symfony/phpunit-bridge": "^3.2",
+        "symfony/phpunit-bridge": "^3.3",
```

Comment:
would this force us to use namespaces in all tests of cmf that use @2.1 of testing component?

Reply:
no, it does not. it just handles both cases, so it enables us to move forward and use the phpunit 6 version with only namespaces if we want to.